floodviz/static/js/map.js

```diff
@@ -230,6 +232,7 @@
 						}
 					});
 					self.linked_interactions.click(selected);
+					ga_send_event('Map', 'drag_select', selected);
```

Comment:
Good Catch. This definitely threw off the label in the GA views when I was checking.